app/backend/app.py

```diff
@@ -449,6 +452,8 @@ async def setup_clients():
     HUGGINGFACE_API_KEY = os.getenv("HUGGINGFACE_API_KEY")
     DEFAULT_MODEL = os.getenv("DEFAULT_MODEL")
 
+    USE_INJECTION_PROTECTION = os.getenv("USE_INJECTION_PROTECTION", "false").lower() == "true"
```

Comment:
Isn't it this way?

```suggestion
    USE_INJECTION_PROTECTION = os.getenv("USE_INJECTION_PROTECTION", "").lower() == "true"
```

Not that it won't work, but everywhere else we do it the other way.

Reply:
fixed